src/session-summaries/the-intersection-observer-api.md

```diff
@@ -0,0 +1,12 @@
+## Intersection Observer Api
+
+This Video introduces the basics of the **Intersection Observer API** and what it can be used for while taking the user through real world examples which were explained carefully. The Intersection Observer API simply watches an element based on some options passed to it during instantiation and calls the callback function also passed to it on instantiation when certain conditions are met.
+
+The Intersection Observer API can be used to trigger various processes such as animations, API calls, etc. It is also prefered over using scroll events because of performance issues although it is not supported by any version of Internet explorer (lol, what is?).
```

Comment:
I love this `(lol, what is?)`.

Add that you can use a Polyfill for old browsers.

Reply:
Hi, I just updated the PR